src/Mailgun/Model/Suppression/Unsubscribe/IndexResponse.php

```diff
@@ -53,10 +69,28 @@ public static function create(array $data)
     }
 
     /**
+     * Get the Unsusbscribe item models from the response
+     * 
+     * @see Mailgun/Model/Suppression/Unsubscribe/Unsubscribe
+     * 
      * @return Unsubscribe[]
      */
     public function getItems()
     {
         return $this->items;
     }
+    
+    /**
+     * Get the total count of Unsusbscribe in index response
+     * 
+     * @see Mailgun/Model/Suppression/Unsubscribe/Unsubscribe
+     * 
+     * @return integer
+     */
+    public function getTotalCount()
+    {
+        $this->totalCount = count($this->items);
```

Comment:
If I call this method 100 times I will run the `count` function 100 times, AKA performance impact. Due the result won't change, you can just set the `totalCount` value inside the constructor

Reply:
Are you sure that the count will not change? For example if I add items manually or I add items (because maybe I want merge a list of mails to unsubscribe and after add more to resent to the API) ?

Reply:
Also, if it doesn't change the value (following your thoughts) it is really more probably call 1000 times to getTotalCount() if it doesnt change the value?

Reply:
IMO given this object is instantiated via other SDK calls when we run an API request (via `public static function create(array $data)` static method) then we can calculate the expected figure when we're creating the object.

In the end, constrictor is a `private` method and this object is mapping data that was returned by the system, so I don't think this is really necessary, and I don't think it will change.

In fact, if you really need to add new / non-covered behaviour, then you can always set-up your own `Decorator` to deal with your specific request, but - again, IMHO - I don't think this update is required here.

Reply:
This class is immutable so nothing will change. 

Im fine with calling count 100 times, the performance impact is minimal compared to the HTTP request to mailgun server that we just did. 

However, it seams weird to have the `$this->totalCount`. That variable is never properly used... 

I wish that the `getTotalCount` functions should be rewritten to "make more sense". Either we just do `return count($this->items);` or we store the value in `$this->totalCount` and never writes to that variable again.